scss/_utilities_floats.scss

```diff
@@ -14,44 +14,52 @@
 
   .u-float-right {
     float: right !important;
+    margin-top: 0;
```

Comment:
This breaks our maxim of having utils do only one thing and for that thing is to concisely described in the util. Is this really needed?